lib/lndclient/LndClient.ts

```diff
@@ -1,39 +1,36 @@
-import grpc from 'grpc';
+import grpc, { Metadata } from 'grpc';
 import fs from 'fs';
-import assert from 'assert';
 
 import Logger from '../Logger';
+import BaseClient from'../BaseClient';
 import errors from './errors';
+import enums from '../constants/enums';
 
-/** A class representing a client to interact with a running lnd instance. */
-class LndClient {
-
-  static statuses = {
-    DISABLED: 'DISABLED',
-    CONNECTION_VERIFIED: 'CONNECTION_VERIFIED', // assuming connection does not remain open
-  };
+interface LndClientConfig {
+  disable: boolean;
+  datadir: string;
+  rpcprotopath: string;
+}
 
-  logger: any;
+/** A class representing a client to interact with a running lnd instance. */
+class LndClient extends BaseClient{
   lightning: any;
-  meta: any;
-  status: any;
+  meta: Metadata | undefined;
 
   /**
    * Create an lnd client.
-   * @param {Object} options - The lnd configuration
+   * @param options - The lnd configuration
    */
-  constructor(options) {
+  constructor(options: LndClientConfig) {
```

Comment:
since the type is `LndClientConfig`, I think we should call the param `config` instead of `options`.